index.html

```diff
@@ -1,37 +1,305 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <link rel="stylesheet" type="text/css" href="style.css">    </head>
-    <body>
-        <div id="header" >
-            <div class="container">
-             
-                       <div class="left-header" style="display: flex; border:1px solid black">
-                            <div class="logo" style="float: left; padding: 10px;;" >
-                                <img src="https://themes.3rdwavemedia.com/delta/bs4/5.0/assets/images/logo/logo.png">
-                            </div>
-                            <div class="logo-text" style="padding-top: 10px;margin-left: 2px;line-height: 50px;" >
-                                <h1>DELTA</h1>
-                            </div>
-                       </div>
-                       <div id="right-header">
-                            <ul class="nav-ul" >
-                                    <li id="contact">features</li>
-                                    <li>how it worls </li>
-                                    <li>FAQ</li>
-                                    <li>story</li>
-                                    <li>pricing</li>
-                                    <li >contact</li>
-                            </ul>     
-                      </div>
-
-                   
-                              
+  <head>
+   
+    <link rel="stylesheet" type="text/css" href="main.css" />
+  </head>
+  <body>
+    <input type="checkbox" id="toggle" />
+    <div id="header">
+      <div class="container">
+        <div class="left-header">
+          <div class="logo" style="padding-top: 15px; padding-right: 2px;">
```

Comment:
let's not use inline-styles as a convention (it's not bad or anything, but it's more convenient/modular/reusable/and making use of sass)

Reply:
this applies to all inline styles